### PR TITLE
Add SLSA3 

### PR DESCRIPTION
**Pull request by rafaelbreno**

### CODE REVIEW

**Review by thomasferrandiz — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -37,6 +47,21 @@ image-build-thin:
 		--load \
 	.
 
+.PHONY: push-image-thin
+push-image-thin:
+	docker buildx build \
```
Comment (brooksn) [inline Makefile:52]:
```suggestion
	docker buildx build \
	$(IID_FILE_FLAG) \
```

File: Makefile [hunk 2]
```diff
@@ -50,6 +75,21 @@ image-build-thick:
 		--load \
 	.
 
+.PHONY: push-image-thick
+push-image-thick:
+	docker buildx build \
```
Comment (brooksn) [inline Makefile:80]:
```suggestion
	docker buildx build \
	$(IID_FILE_FLAG) \
```

**Review by brooksn — CHANGES_REQUESTED** [review]

**Comment by brooksn** [comment]
push-to-prime is ready to be added, along with the vault secrets.

**Review by brooksn — APPROVED** [review]

**Pull request merged**